arangod/Cluster/ClusterMethods.cpp

```diff
@@ -3750,10 +3750,17 @@ arangodb::Result hotBackupCoordinator(ClusterFeature& feature, VPackSlice const
       return result;
     }
 
+    auto releaseAgencyLock = scopeGuard([&]{
+      LOG_TOPIC("52416", INFO, Logger::BACKUP)
+          << "Releasing agency lock with scope guard! backupId: " << backupId;
```

Comment:
I don't think the log level should be "INFO" here, rather DEBUG.

Reply:
Done